### PR TITLE
Awesome/remove workiva logo

### PR DESCRIPTION
**Pull request by jthors**
Why: initial state stuff

### CODE REVIEW

File: iXBRLViewerPlugin.nuspec [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>iXBRLViewerPlugin</id>
+    <version>$version$</version>
+    <authors>mms solutions ag</authors>
+    <description>ixbrl viewer plugin package for deployment</description>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <copyright>Copyright (c) mms solutions ag</copyright>
+    <repository type="git" url="https://github.com/mmssolutionsio/taggingplus" />
```
Comment (christianschlaepfer) [inline iXBRLViewerPlugin.nuspec:10]: I think here we should set the repository of the viewer fork as the viewer is independent from taggingplus.
```suggestion
    <repository type="git" url="https://github.com/mmssolutionsio/ixbrl-viewer" />
```

File: iXBRLViewerPlugin/viewer/src/html/inspector.html [hunk 2]
```diff
@@ -177,8 +177,7 @@ <h3 class="collapsible-header" data-i18n="inspector.associatedFacts">Associated
 
     </div>
     <div class="inspector-foot powered-by">
-        <span data-i18n="inspector.poweredBy">Powered by</span>
-        <a href="https://www.workiva.co.uk/solutions/xbrl-and-ixbrl" target="_blank"><img alt="Workiva logo (opens in new window)" src="../img/workiva.svg" /></a>
+      mms
```
Comment (christianschlaepfer) [inline iXBRLViewerPlugin/viewer/src/html/inspector.html]: Do we need this `<div class="inspector-foot powered-by" />` tag? I would remove it if possible.

File: .gitignore [hunk 3]
```diff
@@ -1 +1,4 @@
 node_modules/*
+package-lock.json
```
Comment (christianschlaepfer) [inline .gitignore]: I believe the package-lock is meant to check in to the repository.
```suggestion
```

**Review by christianschlaepfer — DISMISSED** [review]

**Comment by iappert** [comment]
expected results: link to mmssolutiosn.io. text mms solutions ag. everything else is not acceptable...

File: iXBRLViewerPlugin/viewer/src/html/inspector.html (hunk 2 above)
Comment (iappert) [inline iXBRLViewerPlugin/viewer/src/html/inspector.html]: mms solutions ag with link

**Review by iappert — CHANGES_REQUESTED** [review]

File: iXBRLViewerPlugin.nuspec (hunk 1 above)
Comment (jthors) [reply iXBRLViewerPlugin.nuspec:10]: Good point. @iappert ?

Comment (iappert) [reply iXBRLViewerPlugin.nuspec:10]: yes

**Review by iappert — APPROVED** [review]

**Pull request merged**